### PR TITLE
Solution to the Palindromic Array Problem in Python Langauge

### PR DESCRIPTION
**Pull request by mass-Eve**

### CODE REVIEW

File: Arrays/Python/Palindromic Array/readme.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    return 0
+```
+
+## Performace Status : *Solved*
+
+## Solution Screenshot
+
+![Solution Screenshot](image.png)
+
+# Thank You
```
Comment (dhruvabhat24) [inline Arrays/Python/Palindromic Array/readme.md:38]: Well Explained

**Review by dhruvabhat24 — APPROVED** [review]
Pull Request Accepted

File: Arrays/Python/Palindromic Array/readme.md (hunk 1 above)
Comment (mass-Eve) [reply Arrays/Python/Palindromic Array/readme.md:38]: Thank you very much sir😊😊

Comment (dhruvabhat24) [reply Arrays/Python/Palindromic Array/readme.md:38]: @mass-Eve Refer latest Announcement

**Pull request merged**